brave/browser-laptop
# Torrent name is missing once download is started

The name of the torrent shows up when asking the user to confirm
whether to start the download, but it doesn't show up after the
download has started, which is confusing, especially when coming back
to the tab.

Fixes: #7362


## Test Plan:

1. Click any magnet link from https://codepen.io/ferossity/full/qaezaB/
2. Click "Start Download" to start the torrent
3. Ensure that the torrent title shows up at the top of the page
